src/uct/ib/base/ib_iface.c

```diff
@@ -190,26 +190,26 @@ int uct_ib_iface_is_reachable(const uct_iface_h tl_iface, const uct_device_addr_
     uct_ib_iface_t *iface = ucs_derived_of(tl_iface, uct_ib_iface_t);
     const uct_ib_address_t *ib_addr = (const void*)dev_addr;
     union ibv_gid gid;
-    uint8_t is_global;
+    uint8_t is_global, is_local_ib = 1;
```

Comment:
is_local_ib can be defined as 'int' because unlike is_global, it is not passed by reference to a function